VK/VK/AllGroups/AllGroupsViewController.swift

```diff
@@ -48,6 +33,17 @@ final class AllGroupsViewController: UITableViewController {
 
     // MARK: - Private Methods
 
+    private func fetchGroups(searchName: String) {
+        networkService.fetchGroupsSearch(searchName: searchName) { [weak self] result in
+            guard let self = self else { return }
+            switch result {
+            case let .success(groups): self.groups = groups.response.groups
+            case let .failure(error): print(error)
```

Comment:
```suggestion
            case let .success(groups): 
                      self.groups = groups.response.groups
            case let .failure(error): 
                     print(error)
```
вместо принта алерт (рекомендация)

Reply:
Хорошее замечание!

Reply:
Задам вопрос менторам "как обрабатывать ошибки"